changelog.d/13766.bugfix

```diff
@@ -0,0 +1 @@
+Fix a long-standing bug where the `cache_invalidation_stream_seq` sequence would begin at 1 instead of 2.
```

Comment:
for links: this entry got improved in https://github.com/matrix-org/synapse/commit/ec4dca6064792366959e87b626cc06efc69a5dc6.